recipients_app/lib/view/components/income/income_page.dart

```diff
@@ -25,7 +24,7 @@ class IncomePage extends StatelessWidget {
     return Consumer<CurrentUser>(
       builder: (context, currentUser, child) {
         return Padding(
-          padding: edgeInsetsAll12,
+          padding: const EdgeInsets.all(12),
```

Comment:
Should we change it to one of the defined `app_sizes` or do we keep it like this for now as we will do another round of the redesign?

Reply:
another round of redesign where we will define the spacings. not top prior I would say

Reply:
Sounds good 👍